src/main/kotlin/uk/gov/gdx/datashare/enums/EventType.kt

```diff
@@ -3,4 +3,5 @@ package uk.gov.gdx.datashare.enums
 enum class EventType {
   DEATH_NOTIFICATION,
   LIFE_EVENT,
+  ENTERED_PRISON,
```

Comment:
General question, will we be only receiving entered prison events?

Reply:
We can get LEAVING PRISON or lots of other if we wanted